src/main/java/com/stripe/model/WebhookEndpoint.java

```diff
@@ -15,112 +17,178 @@
-    return retrieve(id, (RequestOptions) null);
+  public WebhookEndpoint update(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format(
+            "%s%s", Stripe.getApiBase(), String.format("/v1/webhook_endpoints/%s", this.getId()));
+    return request(ApiResource.RequestMethod.POST, url, params, WebhookEndpoint.class, options);
   }
 
   /**
-   * Retrieve a webhook endpoint.
+   * You can also delete webhook endpoints via the <a
+   * href="https://dashboard.stripe.com/account/webhooks">webhook endpoint management</a> page of
+   * the Stripe dashboard.
    */
-  public static WebhookEndpoint retrieve(String id, RequestOptions options) throws StripeException {
-    return retrieve(id, null, options);
+  public WebhookEndpoint delete() throws StripeException {
+    return delete((Map<String, Object>) null, (RequestOptions) null);
   }
 
   /**
-   * Retrieve a webhook endpoint.
+   * You can also delete webhook endpoints via the <a
+   * href="https://dashboard.stripe.com/account/webhooks">webhook endpoint management</a> page of
+   * the Stripe dashboard.
    */
-  public static WebhookEndpoint retrieve(String id, Map<String, Object> params,
-      RequestOptions options) throws StripeException {
-    return request(RequestMethod.GET, instanceUrl(WebhookEndpoint.class, id), params,
-        WebhookEndpoint.class, options);
+  public WebhookEndpoint delete(RequestOptions options) throws StripeException {
+    return delete((Map<String, Object>) null, options);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="update">
   /**
-   * Update a webhook endpoint.
+   * You can also delete webhook endpoints via the <a
+   * href="https://dashboard.stripe.com/account/webhooks">webhook endpoint management</a> page of
+   * the Stripe dashboard.
    */
-  public WebhookEndpoint update(Map<String, Object> params) throws StripeException {
-    return update(params, (RequestOptions) null);
+  public WebhookEndpoint delete(Map<String, Object> params) throws StripeException {
+    return delete(params, (RequestOptions) null);
   }
 
   /**
-   * Update a webhook endpoint.
+   * You can also delete webhook endpoints via the <a
+   * href="https://dashboard.stripe.com/account/webhooks">webhook endpoint management</a> page of
+   * the Stripe dashboard.
    */
-  public WebhookEndpoint update(Map<String, Object> params,
-      RequestOptions options) throws StripeException {
-    return request(RequestMethod.POST, instanceUrl(WebhookEndpoint.class, this.id), params,
-        WebhookEndpoint.class, options);
+  public WebhookEndpoint delete(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url =
+        String.format(
+            "%s%s", Stripe.getApiBase(), String.format("/v1/webhook_endpoints/%s", this.getId()));
+    return request(ApiResource.RequestMethod.DELETE, url, params, WebhookEndpoint.class, options);
```

Comment:
15) Delete (standard): Instance method to delete a resource on itself, relying on ID attribute of the instance.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/WebhookEndpoint.java#L182-L193